legion/engine/physics/colliders/convexcollider.cpp

```diff
@@ -107,7 +127,30 @@ namespace legion::physics
 
         //log::debug("Collision FOUND between {} and {}!" , compIDA.read().id, compIDB.read().id);
 
-        physics::PhysicsSystem::penetrationQueries.push_back(manifold.penetrationInformation);
+        //physics::PhysicsSystem::penetrationQueries.push_back(manifold.penetrationInformation);
+
+        //convexCollisionInfo
+        //math::vec3 worldFaceCentroidA = manifold.transformA * math::vec4(convexCollisionInfo.ARefFace.ptr->centroid, 1);
+        //math::vec3 worldFaceNormalA = manifold.transformA * math::vec4(convexCollisionInfo.ARefFace.ptr->normal, 0);
+
+        //math::vec3 worldFaceCentroidB = manifold.transformB * math::vec4(convexCollisionInfo.BRefFace.ptr->centroid, 1);
+        //math::vec3 worldFaceNormalB = manifold.transformB * math::vec4(convexCollisionInfo.BRefFace.ptr->normal, 0);
+
+
+        //math::vec3 worldEdgeAPosition = convexCollisionInfo.edgeRef.ptr ? manifold.transformB * math::vec4(convexCollisionInfo.edgeRef.ptr->edgePosition, 1) : math::vec3();
+        //math::vec3 worldEdgeNormal = convexCollisionInfo.edgeNormal;
+
+        //auto abPenetrationQuery =
+        //    std::make_shared< ConvexConvexPenetrationQuery>(convexCollisionInfo.ARefFace.ptr
+        //        , convexCollisionInfo.BRefFace.ptr, worldFaceCentroidA, worldFaceNormalA, convexCollisionInfo.ARefSeperation, true);
+
+        //auto baPenetrationQuery =
+        //    std::make_shared < ConvexConvexPenetrationQuery>(convexCollisionInfo.BRefFace.ptr, convexCollisionInfo.ARefFace.ptr,
+        //        worldFaceCentroidB, worldFaceNormalB, convexCollisionInfo.BRefSeperation, false);
+
+        //auto abEdgePenetrationQuery =
+        //    std::make_shared < EdgePenetrationQuery>(convexCollisionInfo.edgeRef.ptr, convexCollisionInfo.edgeInc.ptr, worldEdgeAPosition, worldEdgeNormal,
+        //        convexCollisionInfo.aToBEdgeSeperation, false);
```

Comment:
will any of this be used?

Reply:
yes i tried to refactor this part and something when wrong along the way, gonna keep it here for now